src/pflib/FastControlCMS_MMap.cxx

```diff
@@ -140,7 +140,7 @@ class FastControlCMS_MMap : public FastControl {
     l1a_charge.pack();
 
     Periodic pled(periodic(LED_PERIODIC));
-    pled.bx = 10;     // needs tuning
+    pled.bx = 12;     // needs tuning
```

Comment:
I would have thought the following fast command (`l1a_led` below) would be the one that was shifted, did you see the LED pulse now starting at 0?

Reply:
Yeah, now it rises in the first BX
![image](https://github.com/user-attachments/assets/ea886059-77d8-4ef8-9ece-573355703c9a)